Documentation/VSCodeConfiguration.md

```diff
@@ -178,98 +164,154 @@ You can create custom tasks (`.vscode/tasks.json`) to quickly compile Serenity.
                 "clear": true
             }
         },
+        {
+            "label": "build",
+            "type": "shell",
+            "command": "bash",
+                "args": [
+                    "-c",
+                    "Meta/serenity.sh build ${input:arch} ${input:compiler}"
+                ],
+                "options": {
+                    "env": {
+                        "SERENITY_RAM_SIZE": "4G",
+                    }
+                },
+                "problemMatcher": [
+                    {
+                        "base": "$gcc",
+                        "fileLocation": [
+                            "relative",
+                            // FIXME: Clang uses ${input:arch}clang
+                            "${workspaceFolder}/Build/${input:arch}"
+                        ]
+                    },
+                    {
+                        "source": "gcc",
+                        "fileLocation": [
+                            "relative",
+                            // FIXME: Clang uses ${input:arch}clang
```

Comment:
It doesn't look like this FIXME is resolved is it? The build dirs when using clang will be missing the trailing "clang"

Reply:
That's a new fixme, GitHub just marks it red, because vanilla JSON does not have comments, but vscode uses `JSON with comments`